FreeSurfer/scripts/FreeSurferHiresWhite.sh

```diff
@@ -1,6 +1,59 @@
 #!/bin/bash
 
-source "${HCPPIPEDIR}/global/scripts/debug.shlib" "$@" # Debugging functions; also sources log.shlib
+# --------------------------------------------------------------------------------
+#  Usage Description Function
+# --------------------------------------------------------------------------------
+
+script_name=$(basename "${0}")
```

Comment:
Change to `script_name=$(basename -- "${0}")` to not error when pasted into interactive (though it will then say "-bash" is the tool name).  Yes, this edit seems to work on mac, too.